app/javascript/react/components/Calendar/Calendar.tsx

```diff
@@ -1,58 +1,49 @@
 import React from "react";
-import moment from "moment";
 
-import * as S from "./Month.style";
+import {
+  today,
+  getDayNumber,
+  getFullWeeksOfMonth,
+} from "../../utils/datesHelper";
 
+import * as S from "./Calendar.style";
 import { yellow } from "../../assets/styles/colors";
 
-const getFullWeeksOfMonth = (year: number, month: number) => {
-  let start = moment([year, month]).startOf("month").startOf("week");
-  let end = moment([year, month]).endOf("month").endOf("week");
-
-  let weeks = [];
-  while (start <= end) {
-    let week = [];
-    for (let i = 0; i < 7; i++) {
-      week.push(start.clone().add(i, "days").format("YYYY-MM-DD"));
-    }
-    weeks.push(week);
-    start.add(1, "week");
-  }
-  return weeks;
-};
-
-const getDayNumber = (date: string) => {
-  return moment(date).format("D");
-};
-
-const Month = () => {
-  const weeks = getFullWeeksOfMonth(2024, 2);
-  console.log(weeks);
+const Calendar = () => {
+  const weeks = getFullWeeksOfMonth(today.year(), today.month());
+  console.log("🦆", today.year());
+  console.log("🦆🦆", today.month());
+  console.log("🦆🦆🦆", weeks);
 
   return (
     <>
       <S.ThreeMonths>
         <S.Month>
           <S.MonthName>February</S.MonthName>
           {weeks.map((week) => (
-            <S.Week>
+            <S.Week key={week[0]}>
               {week.map((day) => (
-                <S.Day shouldColor={true} color={yellow} isCurrentMonth={true}>
-                  <S.DayNumber shouldDisplay={true}>
+                <S.Day
+                  key={day}
+                  $hasBackground={true}
+                  color={yellow}
+                  $isCurrentMonth={true}
+                >
+                  <S.DayNumber $isVisible={true}>
                     {getDayNumber(day)}
                   </S.DayNumber>
-                  <S.Value shouldDisplay={true}>100</S.Value>
+                  <S.Value $isVisible={true}>100</S.Value>
                 </S.Day>
               ))}
             </S.Week>
           ))}
         </S.Month>
-        <S.Month>
+        {/* <S.Month>
```

Comment:
It was intentional? Now we end up with a monthly view, even the file is named Calendar View. Maybe let's move one month view to separate file and then use it in `CalendarView` 3 times?

Reply:
Well, yes it was. 
It'll still be 3-month view aka `calendar`, but I wanted to leave it commented out until I add logic (there were comments about it). 
Sure, I can extract single Month ;)